### DISCUSSION TITLE
grape pruner horizontal offset

### DISCUSSION THREAD

**Discussion by grm3** [Courseplay/Courseplay_FS22#2047] [General]
in my testing it should be -0.3 (for right of vine)

otherwise no course (in my testing) would ever complete. 

tested with fendt 200 and the electric SKE 50

**Comment by Tensuko** [comment]
On my tests, our default was fine.
You got screenshots?

**Reply by grm3** [reply]
i can do better! a video (will delete in a few days and remove the link)

hope this helps. essentially pruning works on no tractors for me (using the latest courseplay download). 

ALSO, i just noticed the pruner does rows twice, but skips a row? I have it set up to NOT skip rows, of course.

https://grm3.com/auto-delete/fs22_cp_pruner_woes.mp4

**Reply by grm3** [reply]
using a pruner on front and back (one for each side) i get best results at 0.0 offset, but still misses a few vines.

**Reply by Tensuko** [reply]
You did it wrong.
Pls follow the guid of our ingame manual on how the tools supposed to be used!

**Reply by grm3** [reply]
it doesn't matter how i use the pruner, the default courseplay offset does not work. right of vine, left of vine, front or back mount ... it does not matter. it would be best to check where the tool is mounted then create the offset.

![grapes_error](https://user-images.githubusercontent.com/23106060/198724772-daebe3a5-b9a8-4680-b887-c7970fe06be0.png)
![grapes_working](https://user-images.githubusercontent.com/23106060/198724812-dc85f3c7-9708-4d61-9104-77903dbe8724.png)

**Reply by Tensuko** [reply]
Becosue you did NOT read the ingame help.
Working as intended:
![fsScreen_2022_10_28_22_27_48](https://user-images.githubusercontent.com/27344744/198726491-ec8216f7-84e1-4957-ba2a-bf48b3855e4e.png)

![fsScreen_2022_10_28_22_28_42](https://user-images.githubusercontent.com/27344744/198726592-0993195c-12b1-4cf6-8fd4-0a107259f711.png)
